src/TesApi.Web/BatchScheduler.cs

```diff
@@ -834,14 +981,21 @@ private async Task<CloudTask> ConvertTesTaskToBatchTaskAsync(TesTask task, bool
             var batchScriptSasUrl = await this.storageAccessProvider.MapLocalPathToSasUrlAsync($"/{batchScriptPath}");
             var batchExecutionDirectorySasUrl = await this.storageAccessProvider.MapLocalPathToSasUrlAsync($"/{batchExecutionDirectoryPath}", getContainerSas: true);
 
-            var cloudTask = new CloudTask(taskId, $"/bin/sh {batchScriptPath}")
+            //var batchRunScriptPath = "task-run.sh";
```

Comment:
remove?

Reply:
I missed that I hadn't remove these two. I had them here in case it turned out that I'd need them, and in the end, I didn't.